src/main/resources/interface-specification.yml

```
@@ -157,6 +157,19 @@ paths:
                 $ref: '#/components/schemas/Problem'
       operationId: getAttributeById
       description: 'returns the attribute in the registry corresponding to attributeId, if any.'
+  /jobs/attributes/certified/load:
+    parameters:
+      - $ref: '#/components/parameters/CorrelationIdHeader'
+      - $ref: '#/components/parameters/IpAddress'
+    post:
+      summary: Certified attributes upload
+      tags:
+        - attribute
+      responses:
+        '200':
```

More suitable return code when there is no content returned
```suggestion
        '204':
```